index.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+const greetings = require('./lib/greet.js');
+
+//console.log(greetings.greet('maria'));
```

Comment:
That's really weird! I deleted those, and the most recent version I pushed that I have on my laptop doesn't have those comments. I'll fix it 'cause I hate dead code too, thx